### DISCUSSION TITLE
No update on Academic Benefits after approval from GitHub Education - Student Developer Pack

### DISCUSSION THREAD

**Discussion by mohid-mughal** [community/community#132517] [GitHub Education]
### Hi everyone,

Hi,
It has been days (a week probably) since my application for GitHub Education's Student Developer Pack was approved. But i have not received any Academic Benefits yet. Could you update me on when I can access these resources? Or if possible, could you resolve this issue right now?

![image](https://github.com/user-attachments/assets/c8c1178c-99a2-4c7d-8b1f-bf0fcdfc9f01)

**Comment by thiunuwan** [comment]
Log in to GitHub Education: Go to the [GitHub Education page](https://education.github.com/) and log in with your GitHub account. You should see the status of your application and links to the benefits.

Activate Individual Benefits: Some benefits may require additional steps to activate. Go to the [Student Developer Pack page](https://education.github.com/pack) and click on the benefits you're interested in. Follow the instructions provided by each partner.

**Reply by mohid-mughal** [reply]
I haven't been given the access to the academic benefits till now. So the info you provided above isn't currently applicable to me but i will try it for sure, once I get the benefits.

Thanks
